### PR TITLE
Hacktoberfest Leaderboard parser

### PR DESCRIPTION
**Pull request by earlduque**
!hackleaders will pull up the same leaderboard that is displayed on the main hacktoberfest repo

### CODE REVIEW

File: Parsers/Hacktoberfest Leaderboard.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+);
+count.groupBy("user");
+count.addAggregate("SUM", "point");
+count.orderByAggregate("SUM", "point");
+count.query();
+var items = [];
+while (count.next()) {
+  var item = {};
+  item.username = count.user.toString();
+  item.points = parseInt(count.getAggregate("SUM", "point").split(".")[0]);
```
Comment (SapphicFire) [inline Parsers/Hacktoberfest Leaderboard.js:20]: You're likely aware, but you don't use this value anywhere

**Review by SapphicFire — APPROVED** [review]
Looks good to me! Keen to see if you decide to make points transparent, but otherwise looks good

**Comment by earlduque** [comment]
What do you mean transparent

**Comment by SapphicFire** [comment]
As in presently we see the bandings but the actual point values are opaque. Not an issue but that's what I meant by transparent

**Comment by earlduque** [comment]
ohhh, i mean that's the same reason we moved the points thing away from actual numbers. discourages the "doing it for points" thing

**Pull request merged**